colossus/src/main/scala/colossus/protocols/http/Header.scala

```diff
@@ -35,7 +36,7 @@ object HttpVersion {
   case object `1.1` extends HttpVersion("1.1")
 
   def apply(str: String): HttpVersion = {
-    if (str == "HTTP/1.1") `1.1` else if (str=="HTTP/1.0") `1.0` else throw new parsing.ParseException(s"Invalid http version '$str'")
+    if (str == "HTTP/1.1") `1.1` else if (str=="HTTP/1.0") `1.0` else throw new ParseException(s"Invalid http version '$str'")
```

Comment:
I did some checking and I'm almost positive the JVM will intern the string literals, so not much else we can do here.